node/impl/storminer.go

```diff
@@ -49,54 +48,63 @@ import (
 type StorageMinerAPI struct {
```

Comment:
We should consider renaming this to Provider, as the term "Miner" becomes overloaded. It refers both to _any_ process on the miner-side deployment, as well as the specific process that does storage mining.

Reply:
However, this is clearly a follow-up PR, possibly for 1.13 where we bring more consistency to this rearchitecture and we rename even commands and binaries (`lotus-provider`).

Reply:
Noted this in a list with follow-ups, and will mark as resolved, so that we concentrate on outstanding issues for this PR!